node/src/Worker.ts

```diff
@@ -247,11 +245,13 @@ const workerLogger = new Logger('Worker');
 export class Worker<
 	WorkerAppData extends AppData = AppData,
 > extends EnhancedEventEmitter<WorkerEvents> {
-	// mediasoup-worker child process.
-	#child: ChildProcess;
+	/**
+	 * WorkerChannel instance.
+	 */
+	#workerChannel: WorkerChannel;
 
 	// Worker process PID.
-	readonly #pid: number;
+	readonly #pid: number = process.pid;
```

Comment:
This amazes me. Does it mean that we only use the CPU core in which the Node process runs for **all the workers**? Ok, in theory those are native threads so they may use different CPU cores, is it?

Reply:
Correct, threads are not pinned to specific CPU.